test/test-integration/src/test/java/org/wso2am/micro/gw/tests/testCaseBefore/MgwWithNoAPIs.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2am.micro.gw.tests.testCaseBefore;
+
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.wso2am.micro.gw.tests.common.BaseTestCase;
+import org.wso2am.micro.gw.tests.util.TestGroup;
+
+public class MgwWithNoAPIs extends BaseTestCase {
```

Comment:
Whats the purpose of this test. It does not assert anything. This test case always passes

Reply:
It is not a test case, this class defines the what to do before the group.